config/notifiers.go

```diff
@@ -101,14 +101,9 @@ var (
 	// DefaultWechatConfig defines default values for wechat configurations.
 	DefaultWechatConfig = WechatConfig{
 		NotifierConfig: NotifierConfig{
-			VSendResolved: true,
+			VSendResolved: false,
 		},
-		Message:   `{{ template "wechat.default.message" . }}`,
-		APISecret: `{{ template "wechat.default.api_secret" . }}`,
-		ToUser:    `{{ template "wechat.default.to_user" . }}`,
```

Comment:
Why aren't these templated? We do for other notifiers.

Reply:
I must admit I have no clue. @songjiayang any insight?

Reply:
Hi, guys, I don't known the benefit  of templated this fields, I support to delete it  like @simonpasquier pr.

Reply:
I say we leave it in and just update the wechat secrets to the `Secret` type for this PR.

Reply:
Ack @stuartnelson3.  I've rebased the PR to remote the commit on the templated fields.